.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,133 @@
+name: Rollkit Integration Test
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+  workflow_dispatch:
+
+jobs:
+  rollkit-test:
+    name: Test with Rollkit Chain
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    env:
+      DO_NOT_TRACK: true
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: ./go.mod
+          cache: true
+
+      - name: Install Ignite CLI
+        run: |
+          curl -sSL https://get.ignite.com/cli@v28.10.0! | bash
+
+      - name: Start Local DA
+        run: |
+          # clone rollkit repository
+          git clone https://github.com/rollkit/rollkit --depth 1
+          cd rollkit/da/cmd/local-da
+          # start the local da in the background
+          go run . &
+          # capture the background process PID
+          echo "DA_PID=$!" >> $GITHUB_ENV
+          # give it a moment to start
+          sleep 3
+
+      - name: Scaffold Rollkit Chain
+        run: |
+          # scaffold a new chain
+          ignite scaffold chain gm --no-module --skip-git
+          cd gm
+
+          # install rollkit app
+          ignite app install github.com/ignite/apps/rollkit@8dc24c9585058ac1b80ba1d3705f041d4d4a4639
+
+          # add rollkit to the chain
+          ignite rollkit add
+
+      - name: Replace ABCI Module with Current Branch And Prepare Chain
+        run: |
+          # get the path to the current checkout of go-execution-abci
+          CURRENT_DIR=$(pwd)
+          GO_EXECUTION_ABCI_DIR=$CURRENT_DIR
+
+          # enter the gm directory
+          cd gm
+
+          # replace the github.com/rollkit/rollkit module with latest main
+          # TODO(@julienrbrt) this should be replaced with a specific commit hash
+          go mod edit -replace github.com/rollkit/rollkit=github.com/rollkit/rollkit@v0.14.2-0.20250506155912-dccebc0cab3e
```

Comment:
@main works locally, but for some reason gives go.mod parsing errors on CI.
so it is hardcoded for now. We should bump it occasionally here.

I first thought of getting the latest from the go-execution-abci go.mod, but i believe we should strive to test with latest.

Reply:
it think this should do it for CI: https://github.com/rollkit/go-execution-abci/pull/81/commits/f80569d3ff4e0651b3b8ad0181dbf5a6bb133fcf ... let's see.

Reply:
yay!